src/index.html

```diff
@@ -5,6 +5,11 @@
   <title>SohoXI - Angular Components</title>
   <base href="/">
 
+  <!-- prevent render loop from initializing on load -->
```

Comment:
Is this something we should all look at doing?

Reply:
Yeah, before [Eds change](https://github.com/infor-design/enterprise/pull/1098) renderLoop initialized on load. He has now changed that to be optional via a config setting.

`if (this.settings.noAutoStart !== true) {
      this.start();
    }
`

We can set this in index.html before sohoxi.js loads . And then start it later manually in our ### app.component _ngOnInit()_ There is a requestAnimationFrame in the renderLoop that was causing changeDetection to fire. The renderLoop service allows us to call that outside of the Angular zone.

Reply:
@bthharper I can add a note to the change log. Probably should have done that.

Reply:
I think this can wait for ep-ng of 5.x.

Reply:
Okay - might be worth creating the new CHANGELOG section.